### ISSUE TITLE
Mozillaisms and other redundant settings that can be removed.

### ISSUE THREAD

**Issue by Peacock365** [MoonchildProductions/UXP#1610]
Dear developers,

in the following a list of about:config settings that should be removed (or corrected) in my opinion, as of Pale Moon 28.11.0. Please forgive me my ignorance if I made any mistake here.

**Telemetry / crash reporter / Firefox experiments**

_toolkit.telemetry.unified
toolkit.telemetry.server_owner
toolkit.telemetry.server
toolkit.telemetry.reportingpolicy.firstRun
toolkit.telemetry.infoURL
toolkit.telemetry.enabled
toolkit.telemetry.debugSlowSql
devtools.telemetry.tools.opened.version
devtools.telemetry.supported_performance_marks
dom.ipc.plugins.flash.subprocess.crashreporter.enabled
dom.ipc.plugins.reportCrashURL
network.allow-experiments_

**Mozillaisms**

_browser.addon-watch.ignore
devtools.devedition.promo.url
devtools.devices.url
dom.push.serverURL
media.gmp-manger.url
media.gmp-manager.certs.1.commonName
media.gmp-manager.certs.2.commonName
services.sync.fxa.privacyURL
services.sync.fxa.termsURL
services.sync.tokenServerURl_

Again, forgive me my ignorance if I listed legitimate entries here that need to stay that way, this is just a post aimed at listing settings that might be redundant / overseen. I also don't think that Pale Moon / Basilisk still has active telemetry, crash reporter or something, I am sure most of these settings don't do anything anymore, but if that is the case, you might consider removing them to avoid confusion. Thank you for your attention.

**Comment by mattatobin** [comment]
Just stop making irrelevant noise.

**Comment by Peacock365** [comment]
@mattatobin 

How about you clean up redundant about:config entries to avoid confusion, instead of just closing this comment with a snarky remark?

@wolfbeast, I leave this to you, I know Tobin from the forums and I refuse to communicate with him.

**Comment by mattatobin** [comment]
This issue tracker is not for you. It is for us. You can ask your questions at the forum. You have no choice who interacts with you here, like it or not UXP is as much mine as it is Moonchild's or Travis's or athenian200's or Ryan's or anyone else's who helped create and who helps maintain it.

I am telling you that this issue noise is unacceptable, the issue is not solicited, and this is most decently not one issue at all. It is innane rambling and I ask you to stop and head back to the forum to discuss these bits and bobs and WE will decide what and if any single bit gets made into an issue to be resolved.

**Comment by Peacock365** [comment]
Pale Moon is not the only browser I use, is that a problem to you? I would also be more careful to avoid libel like "the criminal Eich" if I were in your shoes, you are just lucky that Eich likely doesn't care about the opinion of a random stranger enough to take any action here. Anyway, not my business, I have no dog in this fight (I am just a user).

Furthermore, I never implied that I make any decision here!? I opened this issue in good faith, and I made it explicit that this is aimed at being a cleanup, I even made it explicit that I do not(!!!) believe that these settings do anything, but they should still be removed for cosmetical(!!!) reasons if not for any other reason. 

And now I would really like to end the conversation with you, because I doubt that anything fruitful will come out of it. Labeling an issue related to cleanup as "invalid", an issue where I never implied that these settings do anything in the first place, also isn't all too helpful.
